recipes/sleapyfaces/meta.yaml

```
@@ -0,0 +1,84 @@
+{% set name = "sleapyfaces" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sleapyfaces-{{ version }}.tar.gz
+  sha256: 071f08edcd5a8ed7a51f9b8482e95c8d1573fabb0976d09bced3c4dcc635a0f7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - hatchling >=1.11.0
+    - hatch >=1.6.0
+    - matplotlib-base >=3.5.0
+    - ffmpeg-python >=0.2.0
+    - hdf5 >=1.12.0
+    - h5py >=3.7.0
+    - numpy >=1.24.0
+    - numexpr >=2.8.0
+    - pytables >=3.6.0
+  host:
+    - python >=3.7
+    - pip >=22.0.0
+    - hatchling >=1.11.0
+    - hatch >=1.6.0
```

```suggestion
    - pip
    - hatchling
```
You only require hatchling without versions upstream